.travis.yml

```diff
@@ -19,4 +21,15 @@ matrix:
       env: TEST_SUITE=lint
     - node_js: "4"
       env: TEST_SUITE=coveralls
-script: npm run $TEST_SUITE
+    - node_js: "4"
+      env: TEST_SUITE=saucelabs
+  allow_failures:
+    - env: TEST_SUITE=saucelabs
+fast_finish: true
+cache:
+  directories:
+    - node_modules
+before_script:
+  - npm install -g grunt-cli
+script:
+  - ./travis.sh
```

Comment:
here we go! 

https://github.com/evilaliv3/elliptic/commit/8e8a568e03a326f3814ac2a2e0a573a958c15698